src/es/ikigaimangas/src/eu/kanade/tachiyomi/extension/es/ikigaimangas/IkigaiMangas.kt

```diff
@@ -313,7 +318,7 @@ class IkigaiMangas : HttpSource(), ConfigurableSource {
         private const val BASE_URL_PREF = "overrideBaseUrl"
         private const val BASE_URL_PREF_TITLE = "Editar URL de la fuente"
         private const val BASE_URL_PREF_SUMMARY = "Para uso temporal, si la extensión se actualiza se perderá el cambio."
-        private const val DEFAULT_BASE_URL_PREF = "defaultBaseUrl"
+        private const val DEFAULT_BASE_URL_PREF = "https://lectorikigai.acamu.net"
```

Comment:
don't change this, it's the preference key

Reply:
Hardcode `https://ikigaimangas.com` [here](https://github.com/keiyoushi/extensions-source/blob/a3e01980d4516f1c558c063389c2e92b88219323/src/es/ikigaimangas/src/eu/kanade/tachiyomi/extension/es/ikigaimangas/IkigaiMangas.kt#L50) as well

Reply:
thanks! still learning and trying to understand the codebase better...

Reply:
I missed this part; please hardcode the url [here](https://github.com/keiyoushi/extensions-source/blob/3ac2800dc2856312cb56aeed140a2b175e0e4c02/src/es/ikigaimangas/src/eu/kanade/tachiyomi/extension/es/ikigaimangas/IkigaiMangas.kt#L53) too